src/re2.h

```diff
@@ -204,10 +204,14 @@ struct PlayerEntity : Entity
     int16_t Sce_flg;
     uint8_t Em_set_flg;
     uint8_t Model_type;
-    int field_1D0;
-    int field_1D4;
-    int field_1D8;
-    int field_1DC;
+    uint16_t Damage_flg;
+    uint8_t Damage_no;
+    uint8_t Damage_cnt;
+    uint16_t Sce_free0;
+    uint16_t Sce_free1;
+    uint16_t Sce_free2;
+    uint16_t Sce_free3;
+    uint16_t Spl_flg;
```

Comment:
You replaced 16 bytes with only 14. This means the following fields are not the correct offset.

Reply:
I added missing uint16_t `Parts0_pos_y`